packages/smooth_app/lib/bottom_sheet_views/user_contribution_view.dart

```diff
@@ -0,0 +1,294 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+import 'package:smooth_app/generated/l10n.dart';
+import 'package:smooth_app/functions/launchURL.dart';
+import 'package:smooth_ui_library/buttons/smooth_simple_button.dart';
+import 'package:smooth_ui_library/dialogs/smooth_AlertDialog.dart';
+import 'package:smooth_ui_library/widgets/smooth_listTile.dart';
+
+Launcher launcher = Launcher();
+
+class UserContributionView extends StatelessWidget {
+  const UserContributionView(this._scrollController, {this.callback});
+
+  final ScrollController _scrollController;
+  final Function callback;
+
+  static final List<Color> _colors = <Color>[
+    Colors.black87,
+    Colors.green.withOpacity(0.87),
+    Colors.deepOrangeAccent.withOpacity(0.87),
+    Colors.redAccent.withOpacity(0.87),
+  ];
+  static const Color _COLOR_DEFAULT = Colors.black26;
+
+  static Color getColor(final int index) => _colors[index] ?? _COLOR_DEFAULT;
+
+  @override
+  Widget build(BuildContext context) {
+    return Material(
+      child: Container(
+        height: MediaQuery.of(context).size.height * 0.9,
+        child: Stack(
+          children: <Widget>[
+            Column(
+              mainAxisSize: MainAxisSize.max,
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: <Widget>[
+                Container(
+                  height: MediaQuery.of(context).size.height * 0.9,
+                  child: ListView(
+                    controller: _scrollController,
+                    shrinkWrap: true,
+                    scrollDirection: Axis.vertical,
+                    children: <Widget>[
+                      Container(
+                        padding: const EdgeInsets.symmetric(horizontal: 20.0),
+                        margin: const EdgeInsets.only(top: 20.0, bottom: 24.0),
+                        child: Text(
+                          S.of(context).contribute,
+                          style: Theme.of(context).textTheme.headline1,
+                        ),
+                      ),
+                      Column(
+                        /// NGO FEE
+                        /// https://stackoverflow.com/questions/46505660/how-to-create-a-google-play-console-developper-account-for-our-ngo
```

Comment:
Not sure why that comment is here :relaxed:

Reply:
:) A couple of thoughts from me. I'll probably make an issue about it in the next few days.